engine/installation/linux/docker-ee/ubuntu.md

```diff
@@ -159,12 +161,11 @@ from the repository.
     $ sudo apt-get install docker-ee
     ```
 
-    > **Warning**:
-    > If you have multiple Docker repositories enabled, installing
+    > **Warning**: If you have multiple Docker repositories enabled, installing
     > or updating without specifying a version in the `apt-get install` or
     > `apt-get update` command will always install the highest possible version,
     > which may not be appropriate for your stability needs.
-    {:.warning}
+    {:.warning-vanilla}
```

Comment:
I think this whole note might show up bold with the `{: . warning-vanilla}` tag. If you use that one, you need to have something acting as a title for it ...